src/deploy/firestore/release.ts

```diff
@@ -1,19 +1,26 @@
 import * as _ from "lodash";
 
 import { RulesDeploy, RulesetServiceType } from "../../rulesDeploy";
+import { Options } from "../../options";
+import { FirebaseError } from "../../error";
 
 /**
  * Releases Firestore rules.
  * @param context The deploy context.
  * @param options The CLI options object.
  */
-export default async function (context: any, options: any): Promise<void> {
+export default async function (context: any, options: Options): Promise<void> {
```

Comment:
This is the only file where I am actually using `Options`, `Config`, and `FirebaseConfig` types. It's a proof of concept.